inc/domainrecordtype.class.php

```diff
@@ -42,46 +42,225 @@ class DomainRecordType extends CommonDropdown
       [
          'id'        => 1,
          'name'      => 'A',
-         'comment'   => 'Host address'
+         'comment'   => 'Host address',
+         'fields'    => [],
       ], [
          'id'        => 2,
          'name'      => 'AAAA',
-         'comment'   => 'IPv6 host address'
+         'comment'   => 'IPv6 host address',
+         'fields'    => [],
       ], [
          'id'        => 3,
          'name'      => 'ALIAS',
-         'comment'   => 'Auto resolved alias'
+         'comment'   => 'Auto resolved alias',
+         'fields'    => [],
       ], [
          'id'        => 4,
          'name'      => 'CNAME',
          'comment'   => 'Canonical name for an alias',
+         'fields'    => [],
       ], [
          'id'        => 5,
          'name'      => 'MX',
-         'comment'   => 'Mail eXchange'
+         'comment'   => 'Mail eXchange',
+         'fields'    => [
+            [
+               'label'       => 'Priority',
+               'placeholder' => 'eg. 10',
```

Comment:
as we provide strings directly in code and not in dropdown values, maybe we can directly translate them so (aka use `| __(...)`)

Reply:
These strings will be loaded in dropdown values, so they will be translatable.

We can translate their default value (in order to have defaults translated in GLPI default lang), but it requires to move all this `$knowtypes` declaration into a method. Indeed, we cannot use the `__()` function in a property declaration.

Reply:
I guess we should not have strings to translate in most of placeholders; just data is enough and more clear for almost all of present cases.